src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -237,8 +238,16 @@ protected boolean setupAndConsumeRecipeInputs(Recipe recipe) {
             recipe.matches(true, importInventory, importFluids);
     }
 
+    protected int[] calculateOverclock(int EUt, int duration) {
+        if (this.overclockVoltage == -1) {
```

Comment:
This would feel slightly less magical if we were just testing for a value > 0

Reply:
you mean >= 0
0 is that special case value of OFF
-1 is the infamous not configured value

Reply:
You mean >= 0

0 means OFF
-1 is the contentious not configured value